pkg/playground/react-patterns.js

```diff
@@ -22,8 +22,8 @@ require([
     "react",
     "performance/dialog-view",
     "playground/react-demo-dialog",
-    "base1/cockpit-components",
-], function(cockpit, React, dialog_pattern, demo_dialog, components) {
+    "base1/cockpit-components-terminal",
+], function(cockpit, React, dialog_pattern, demo_dialog, cockpit_terminal) {
```

Comment:
> I would suggest a more measured approach, where if AMD modules are not in use they all go into the global namespace. But if AMD is in use they go into individual modules. What do you think?

Makes sense, but I think this would be good in a follow-up.